lib/developer_portal/app/controllers/developer_portal/base_controller.rb

```diff
@@ -25,4 +25,11 @@ def finish_signup_for_paid_plan
 
     redirect_to edit_payment_details_path
   end
+
+  def filter_readonly_params(params, resource_class)
+    return unless params
```

Comment:
is there a reason for returning `nil` here? Shouldn't it be `[]` or something? Then later you have 
```
    def account_editable_params(account_params)
      return {} unless account_params
```

It just feels a little inconsistent. Maybe it makes sense but I have to ask.

Reply:
Done: https://github.com/3scale/porta/pull/3997/commits/767e100b92dea8ca71e5d24ab05155632b37e04a